ProxyHandler.class.php

```diff
@@ -123,6 +127,20 @@ function __construct($options)
             $this->_follow_location = true;
             $this->setCurlOption(CURLOPT_FOLLOWLOCATION, true);
         }
+        if (isset($options['forceHttpVersion'])) {
+            $v = $options['forceHttpVersion'];
+            switch ($options['forceHttpVersion']) {
+            case 'HTTP/1.0':
+                $this->setCurlOption(CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_1_0);
+                break;
+            case 'HTTP/1.1':
+                $this->setCurlOption(CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_1_1);
+                break;
+            default:
```

Comment:
How about `CURL_HTTP_VERSION_2_0`, `CURL_HTTP_VERSION_2TLS`, and `CURL_HTTP_VERSION_2_PRIOR_KNOWLEDGE`